probes/pkg/dns/metrics.go

```diff
@@ -14,24 +14,36 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package metrics
+package dns
```

Comment:
Hmm - github suggests you move pkg/metrics/metrics to this one. Is that true? Or is it a copy?

Reply:
OK - never mind, I see there are two files now.